lib/hello_elixir.ex

```diff
@@ -0,0 +1,18 @@
+defmodule HelloElixir do
+  @moduledoc """
+  Documentation for `HelloElixir`.
+  """
+
+  @doc """
+  Hello world.
+
+  ## Examples
+
+      iex> HelloElixir.hello()
+      "heil elixir!"
+
+  """
+  def hello do
+    "heil elixir!"
```

Comment:
In der deutschen Sprache werden Substantive mit einem Großbuchstaben begonnen.

Reply:
Fixed grammar